cmake/msys/msys_helper.sh

```diff
@@ -90,33 +90,33 @@ if [ $? -ne 0 ]; then
 	if [ $? -ne 0 ]; then
         	err "ERROR: Could not build/install fltk -- Zyn needs this.  Exiting."
 	fi
-	
-#	ln -s $mingw_root/usr/local/bin/fluid.exe $mingw_root/bin/fluid.exe 	
+
+#	ln -s $mingw_root/usr/local/bin/fluid.exe $mingw_root/bin/fluid.exe
 	popd
 else
-	warn "  - Skipping, fluid binary already exists" 
+	warn "  - Skipping, fluid binary already exists"
 fi
 
 info "Downloading and building libogg $oggver"
 
-if [ ! -e $mingw_root/lib/libogg.dll.a ]; then 
-	wget http://downloads.xiph.org/releases/ogg/libogg-$oggver.tar.xz -O $HOME/libogg-source.tar.xz
+if [ ! -e "$mingw_root/lib/libogg.dll.a" ]; then
+	wget http://downloads.xiph.org/releases/ogg/libogg-$oggver.tar.xz -O "$HOME/libogg-source.tar.xz"
```

Comment:
>  Because a few lines before: oggver="1.3.2", so it's okay. :)

And it knows there will never be a space in the value?

 * If `"$QT5"` were really so dangerous in an `if` statement, how come Ubuntu 12.04 and 14.04 have never had a problem with it?